### DISCUSSION TITLE
Git How to go back to a previous release

### DISCUSSION THREAD

**Discussion by 1s2d3i4i** [community/community#48267] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How can I get to a previous relase in Github? I can commit push pull and what ever, but how can I come back to a previous release I can't find. For Debugging purposes it is important, to be able to access previous releases. How can I do this with GIT.
Thanks in advance

**Comment by spenserblack** [comment]
Locally, `git checkout v1.2.3`, where `v1.2.3` is the git tag associated with the release.

But if you're debugging between versions, you'll probably want to use [`git bisect`](https://git-scm.com/docs/git-bisect).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @1s2d3i4i ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
